spec/features/user_can_read_messages_spec.rb

```diff
@@ -0,0 +1,16 @@
+feature "User can read messages" do 
+    let(:sender) {create(:user, email: "sender@mail.com", password:"wderfhujikki", name:"pauline") }
+    let(:receiver) {create(:user, email: "receiver@mail.com",password:"hdjkkkklkk", name:"ali") }
+
+    describe "sender sends message and user logs in" do  
+        before do
+            sender.send_message(receiver, "Lorem ipsum...", "Subject")
+            login_as(receiver, scope: :user)
+            visit mailbox_inbox_path
+        end
+
+        it "receiver checks inbox" do
+            expect(page).to have_content "Lorem ipsum..."
+        end
```

Comment:
Could a mailbox helper be used to check the content of the inbox directly?

Reply:
You can check number of messages in mailbox, but I don't know if we can check content